setup.py

```diff
@@ -12,40 +12,34 @@ def get_cuda_version():
     except Exception as e:
         return 'no_cuda'
 
-def get_install_requires(cuda_version):
```

Comment:
The func was introduced in #259, why remove it?

Reply:
And even this commit #259 is buggy. It actually wants to print a warning when the version is not 12.4. but actually it prints a warning when the version is equal to 12.4...😢

Reply:
The way it checks the cuda version  is also incorrect. It checks the version of the system cuda rather than the cuda bundled with the PyTorch installation. So the get_cuda_version func could be removed in a future commit.